neo4j/src/test/java/org/hibernate/ogm/datastore/neo4j/test/index/Neo4jIndexTest.java

```diff
@@ -0,0 +1,216 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+package org.hibernate.ogm.datastore.neo4j.test.index;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Index;
+import javax.persistence.Table;
+
+import org.hibernate.SessionFactory;
+import org.hibernate.ogm.datastore.neo4j.impl.Neo4jIndexSpec;
+import org.hibernate.ogm.datastore.neo4j.test.mapping.Neo4jJpaTestCase;
+import org.hibernate.ogm.datastore.neo4j.utils.Neo4jTestHelper;
+import org.hibernate.ogm.datastore.spi.DatastoreProvider;
+import org.hibernate.ogm.exception.EntityAlreadyExistsException;
+import org.hibernate.ogm.utils.TestForIssue;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import org.neo4j.graphdb.Label;
+
+import static org.fest.assertions.Assertions.assertThat;
+import static org.hibernate.ogm.datastore.neo4j.test.util.ExceptionHelper.extract;
+
+/**
+ * @author The Viet Nguyen
+ */
+@TestForIssue(jiraKey = "OGM-1462")
+public class Neo4jIndexTest extends Neo4jJpaTestCase {
```

Comment:
Could you add a test where the index has applied on column, where the **column** name it is different from **field** name?

Reply:
interesting case, I sure will add it

Reply:
Thank you

Reply:
I have checked, this case is already covered by hibernate-core